src/semver.ts

```diff
@@ -66,10 +66,13 @@ export async function bumpVersion(
   }
 
   if (opts.suffix) {
+    const date = new Date();
+    // YYMMDD-HHMMSS: 2024819-135530
+    const dateStr = `${date.getFullYear()}${date.getMonth()}${date.getDate()}-${date.getHours()}${date.getMinutes()}${date.getSeconds()}`;
```

Comment:
can we use `padStart` to ensure we have a leading zero for these, so they can be sorted/scanned more easily?

Reply:
3c38a835b9e164f6aee56219949da600b363dfdb ~> `20240919-140954`

Reply:
perfect!